docs/user-guide/browser-profiles.md

```diff
@@ -47,8 +47,8 @@ Browser profiles don't just affect websites! Any of Brave's settings (available
 
 Sometimes websites will log users out or expire cookies or login sessions after a period of time. In these cases, when crawling the browser profile can still be loaded but may not behave as it did when it was initially set up.
 
-To update the profile, go to the profile's details page and press the _Edit Browser Profile_ button to load and interact with the sites that need to be re-configured. When finished, press the _Save Browser Profile_ button to return to the profile's details page.
+To update the profile, go to the profile's details page and press the _Configure Browser Profile_ button to load and interact with the sites that need to be re-configured. When finished, press the _Save Browser Profile_ button to return to the profile's details page. Profiles are automatically backed up on save.
```

Comment:
@Shrinks99 @tw4l feel free to directly reword that last sentence if necessary

Reply:
Yes! I'd change to:

```suggestion
To update the profile, go to the profile's details page and press the _Configure Browser Profile_ button to load and interact with the sites that need to be re-configured. When finished, press the _Save Browser Profile_ button to return to the profile's details page. Profiles are automatically backed up on save if replica storage locations are configured.
```

Reply:
Thought about this, but also for most users (on our hosted instance) this is true?  Maybe not worth thinking about too much, doubt people will have trouble with it either way!  Do what y'all think is best & most accurate!